src/Fixer/Whitespace/StatementIndentationFixer.php

```diff
@@ -496,36 +494,6 @@ private function getLineIndentationWithBracesCompatibility(Tokens $tokens, int $
         return $regularIndent;
     }
 
-    private function isCommentForControlSructureContinuation(Tokens $tokens, int $index): bool
```

Comment:
I see that all tests are green and there even new ones, but I need to ask: why it had to be removed? Wasn't it here for purpose?

Reply:
It was partly for the behavior we can see changed in `tests/Fixer/Basic/BracesFixerTest.php`.

I was trying to find some other reason, but failed. So I went with the fact tests are green and the fix is done, and if that introduce new bug we will have it reported and work on that (we can always restore this method.